### ISSUE TITLE
Security hotspot link for Sq 8.5

### ISSUE THREAD

**Issue by rockmagic** [mc1arke/sonarqube-community-branch-plugin#276]
**Describe the bug**

In sq 8.5 security hotspots are not listed among all issues, so the link "View in SonarQube" in MR comment leads to empty issues page with notice: 
> Cannot open selected issue, as it's not part of the initial 1000 loaded issues.

**To Reproduce**
1. Set up pull-request decoration for GitLab
2. Run scanner for merge request with security problem, for example hardcode something like `password = "Password"`
3. Go to Merge request, find the comment for this line, clink link "View in SonarQube"

Incorrect link:
```
https://sq.domain.name/project/issues?id=my%3Aproject&pullRequest=123&issues=AXX-ZP-9j1WiQQuk1P3v&open=AXX-ZP-9j1WiQQuk1P3v
```

**Expected behavior**
Link should head to Security Hotspots page in SonarQube

Corrent link:
```url
https://sq.domain.name/security_hotspots?id=my%3Aproject&pullRequest=123&hotspots=AXX-UssNj1WiQQuk1If0
```

**Screenshots**

![image](https://user-images.githubusercontent.com/14000671/100222359-0346c180-2f2b-11eb-905b-e3730892564a.png)

**Software Versions**
 - SonarQube Version: Version 8.5 (build 38104)
 - Plugin Version: 1.6.0

**Comment by mc1arke** [comment]
Can you provide a screenshot of the reported issue in GitLab please?

**Comment by rockmagic** [comment]
Sure, here's the discussion thread created by PR decoration:

![image](https://user-images.githubusercontent.com/14000671/100453072-65d6c380-30cb-11eb-8940-76c7195c6940.png)

**Comment by rockmagic** [comment]
https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/d5232b4c15751432b57dcf931f2673ccb48f8c91/src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java#L119

Maybe `AnalysisDetails::getIssueUrl` should receive whole issue object, or additional (optional) parameter to discover issue type.

**Comment by mc1arke** [comment]
Released in v1.9.0 of the plugin

**Comment by mc1arke** [comment]
Backported to 1.8.1 of the plugin for Sonarqube 8.9 support.
